cmd/runner/executetest.go

```diff
@@ -211,15 +212,15 @@ func executeTest(ctx context.Context, plan *TestPlan) bool {
 				// Prepare the test command
 				command := []string{"/probe"}
 				arguments := []string{
-					"--url", test.Endpoint,
-					"--timeout", test.Timeout.String(),
-					"--expected-status", strconv.Itoa(test.StatusCode),
+					pf.Flagify(pf.ArgUrl), test.Endpoint,
```

Comment:
See comment in `pkg/probeflags/probeflags.go`:
```suggestion
					pf.Flagify(pf.ArgURL), test.Endpoint,
```